src/fuse.js

```diff
@@ -10,7 +10,10 @@ define([
     util
 ) {
     var FUSE_CONFIG = {
-        keys: ['label', 'name', 'absolutePath', 'hashtagPath'],
+        keys: ['displayPath'],
+        tokenize: true,
+        tokenSeparator: /\/+/g,
+        threshold: 0.4,
```

Comment:
the default threshold is 0.6, but tokenization changes the scores that come back. (noticed because one of the atwho tests failed) I don't see any reason not to limit this more unless someone reports that it isn't returning the appropriate results